File: 1_two_sum.c
int* twoSum(int* nums, int numsSize, int target,int* returnSize) { 
{
    int left = 0, right = 0;
    int* result = (int*)malloc(sizeof(int) * 2);
    for ( left = 0; left < numsSize; left++)
    {
        for (right = left + 1; right < numsSize; right++)
        {
            if (nums[left] + nums[right] == target)
            {
                result[0] = left;
                result[1] = right;
                *returnSize = 2;
                return result;
            }
        }
        
    }
    return 0;
}

File: 1_two_sum.py
from typing import List

# 两层for循环暴力破解，还可以用滑动窗口，二刷时学习
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        left , right = 0 , 0
        for left in range(0,len(nums)):
            for right in range(left+1,len(nums)):
                if(nums[left] + nums[right] == target):
                    return [left,right]


        return 0